src/Opdex.Platform.WebApi/Validation/NetworkAddressValidator.cs

```diff
@@ -17,6 +17,18 @@ public override bool IsValid(ValidationContext<T> context, Address value)
     protected override string GetDefaultMessageTemplate(string errorCode) => "{PropertyName} must be valid address.";
 }
 
+public class NetworkAddressStringValidator<T> : PropertyValidator<T, string>, INetworkAddressValidator
+{
+    public override string Name => "NetworkAddress";
+
+    public override bool IsValid(ValidationContext<T> context, string value)
+    {
+        return Address.TryParse(value, out var address) && address != Address.Empty && address != Address.Cirrus.ToString();
```

Comment:
depending on where this is used, CRS might be problematic

Reply:
We'll create a separate validator `NetworkAddressOrCrsValidator` for that. Will remove the useless `.ToString()` here, just seen that.